File: server/app/model/connect.ts
import pgp from 'pg-promise';

const { DB_ID, DB_PASSWORD, DB_HOST } = process.env;
const connection = {
  host: DB_HOST,
  port: 5432,
  database: 'database',
  user: DB_ID,
  password: DB_PASSWORD,
  max: 30
}
const db = pgp(connection);

export default db;

File: taskserver/app/model/connect.py
import psycopg2 as pg2
import os

def get_connection():
  connect = pg2.connect(host='db',
                    database='test',
                    user=os.getenv('DB_ID'),
                    password=os.getenv('DB_PASSWORD'),
                    port='5432')
  return connect
